### PR TITLE
fix(docs/landing-page): Landing page layout breaks on small devices

### PR DESCRIPTION
**Pull request by matthiashader**
## 💡 What is the current behavior?
- Some sections are breaking on mobile devices
- Overlapping content

GitHub Issue Number: #N/A, [IX-974]

## 🆕 What is the new behavior?

- Section made responsivenes
- improved general sections

## 🏁 Checklist

A pull request can only be merged if all of these conditions are met (where applicable):

- [ ] 🦮 Accessibility (a11y) features were implemented
- [ ] 🗺️ Internationalization (i18n) - no hard coded strings
- [ ] 📲 Responsiveness - components handle viewport changes and content overflow gracefully
- [ ] 📄 Documentation was reviewed/updated (`pnpm run docs`)
- [ ] 🧪 Unit tests were added/updated and pass (`pnpm test`)
- [ ] 📸 Visual regression tests were added/updated and pass ([Guide](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing))
- [ ] 🧐 Static code analysis passes (`pnpm lint`)
- [ ] 🏗️ Successful compilation (`pnpm build`, changes pushed)

### CODE REVIEW

File: packages/documentation/src/pages/index.tsx
Comment (danielleroux) [inline packages/documentation/src/pages/index.tsx:1]: Why did you change the complete file?

**Review by danielleroux — CHANGES_REQUESTED** [review]

File: packages/documentation/src/pages/index.tsx
Comment (matthiashader) [reply packages/documentation/src/pages/index.tsx:1]: Resolved

**Review by danielleroux — APPROVED** [review]

File: packages/documentation/src/pages/index.tsx [hunk 1]
```diff
@@ -602,11 +604,6 @@ function InformationCards() {
   return (
     <div>
       <div className={clsx(styles.container, styles.information_cards)}>
-        <img
-          src={useBaseUrl('/img/Screen_09_background_image.png')}
-          alt=""
-          className={styles.information_cards_background}
```
Comment (nuke-ellington) [inline packages/documentation/src/pages/index.tsx:608]: Removing the class breaks the styling. Also the corresponding CSS class is not used anywhere else so we should remove it from the stylesheet.

File: packages/documentation/src/pages/index.module.scss [hunk 2]
```diff
@@ -351,11 +350,10 @@
     min-height: 100vh;
     background-color: #000028;
 
+
```
Comment (nuke-ellington) [inline packages/documentation/src/pages/index.module.scss:353]:
```suggestion

```

File: packages/documentation/src/pages/index.tsx (hunk 1 above)
Comment (nuke-ellington) [reply packages/documentation/src/pages/index.tsx:608]: Screenshot 2024-07-19 at 10 14 23

**Review by nuke-ellington — CHANGES_REQUESTED** [review]
Additionally the icons get jumbled up when resizing.
Screenshot 2024-07-19 at 10 22 50

**Review by danielleroux — CHANGES_REQUESTED** [review]
Looks broken
image

**Review by danielleroux — DISMISSED** [review]

**Pull request closed without merging**